server/sequelize.js

```diff
@@ -5,6 +5,15 @@ import EncryptedField from "sequelize-encrypted";
 
 const isProduction = process.env.NODE_ENV === "production";
 
+const isPGSqlEnabled =
+  typeof process.env.PGSSLMODE === "undefined" ||
+  process.env.PGSSLMODE === null ||
+  process.env.PGSSLMODE !== "disable";
+
+const isEncryptedConnection = isProduction && isPGSqlEnabled;
+
+console.log(isEncryptedConnection);
```

Comment:
I think checking for the explicit disable of `process.env.PGSSLMODE !== "disable"` should be enough, no?

Reply:
Pls remove `console.log`